igvc_navigation/launch/pather.launch

```diff
@@ -3,7 +3,7 @@
 <!-- map.launch -->
 <launch>
     <!-- Waypoints -->
-    <include file="$(find igvc_navigation)/launch/waypoint_source.launch" />
+    <!-- <include file="$(find igvc_navigation)/launch/waypoint_source.launch" /> -->
```

Comment:
so this is a simulation thing that breaks IRL. If we want to restructure things so that this is not an issue now is the time for that.

Reply:
I agree. A simple workaround would be to include a launch file flag, `simulation`, that doesn't load IRL waypoints when set to true.

Let me know if you think this would be a good approach.

Reply:
yes, let's go with that for now